java-client/src/main/java/org/scassandra/http/client/BatchStatement.java

```diff
@@ -0,0 +1,57 @@
+package org.scassandra.http.client;
+
+public final class BatchStatement {
```

Comment:
This name is initially slightly confusing, since it could be interpreted as: 'A statement containing statements' (what a BatchStatement is in the java-driver) or 'A statement belonging to a batch' (the latter of which is true I think).  A better name might be good, but I can't think of one at the top of my head.